### PR TITLE
Adding CAP users with special permissions

### PR DESCRIPTION
**Pull request by sorinstelian**
Closes https://helpdesk.eaudeweb.ro/issues/8226 https://helpdesk.eaudeweb.ro/issues/8051

For https://helpdesk.eaudeweb.ro/issues/8051:
All country profile endpoints allow read-only access for any authenticated user; special permission needed to be added only for aggregation ODP & MT endpoints (inc. destruction).

### CODE REVIEW

File: ozone/core/api/views.py [hunk 1]
```diff
@@ -422,7 +434,7 @@ class GroupSubstanceViewSet(ReadOnlyMixin, viewsets.ModelViewSet):
 
     queryset = Group.objects.all()
     serializer_class = GroupSubstanceSerializer
-    # Allows unauthenticatd GET requests
+    # Allows unauthenticated GET requests
```
Comment (melish) [inline ozone/core/api/views.py:437]: Is 'unauthenticated' the proper word? Or rather, 'unprivileged'?

File: ozone/core/api/views.py [hunk 2]
```diff
@@ -1653,7 +1676,8 @@ class SubmissionInfoViewSet(viewsets.ModelViewSet):
     obligation_types = None
     serializer_class = SubmissionInfoSerializer
     permission_classes = (
-        IsAuthenticated, IsSecretariatOrSamePartySubmissionRelated,
+        IsAuthenticated,
+        IsSecretariatOrSamePartySubmissionRelated,
```
Comment (melish) [inline ozone/core/api/views.py:1680]: CAP users should be able to see the SubmissionInfo, too

File: ozone/core/api/views.py [hunk 3]
```diff
@@ -753,7 +765,8 @@ class AggregationViewSet(viewsets.ReadOnlyModelViewSet):
     )
     ordering = ("-reporting_period__start_date", "party", "group")
     permission_classes = (
-        IsAuthenticated, IsSecretariatOrSamePartyAggregation,
+        IsAuthenticated
```
Comment (melish) [inline ozone/core/api/views.py:768]: File "/var/local/ozone/ozone/core/api/views.py", line 769, in AggregationViewSet
    & (IsSecretariatOrSamePartyAggregation | IsMobileApp)
TypeError: unsupported operand type(s) for |: 'type' and 'type'

File: ozone/core/api/views.py (hunk 2 above)
Comment (sorinstelian) [reply ozone/core/api/views.py:1697]: Yes, they will be able to see it due to Submission.has_read_rights_for_party(user) being called.

File: ozone/core/api/views.py (hunk 1 above)
Comment (sorinstelian) [reply ozone/core/api/views.py:437]: Yes, it allows read-only access to group-substance data even for unauthenticated users. I think this is currently being used by the Drupal site.

**Pull request merged**